src/client/wieldmesh.cpp

```diff
@@ -313,14 +313,17 @@ static scene::SMesh *createSpecialNodeMesh(Client *client, MapNode n,
 
 	if (n.getParam2()) {
 		// keep it
+	} else if (f.drawtype == NDT_TORCHLIKE) {
+		n.setParam2(1);
```

Comment:
So torches are always supposed to be on-floor even if they have a paramtype set
signs only if they _don't_ have a paramtype

why the difference?